framework/src/application/application.ts

```diff
@@ -470,6 +470,9 @@ export class Application {
 				getSchema: {
 					handler: () => this._node.actions.getSchema(),
 				},
+				getNodeStatusAndConstants: {
```

Comment:
`getNodeInfo` is fine?
Also, if the previous one is not used anymore, then lets remove it

Reply:
Removed and Renamed.